pkg/apis/core/types.go

```diff
@@ -3504,6 +3506,21 @@ type ServiceSpec struct {
 	// cluster (e.g. IPv6 in IPv4 only cluster) is an error condition and will fail during clusterIP assignment.
 	// +optional
 	IPFamily *IPFamily
+
+	// topologyKeys is a preference-order list of topology keys which
+	// implementations of services should use to preferentially sort endpoints
+	// when accessing this Service. Topology keys must be valid label keys and
+	// at most 16 keys may be specified, and can't be used with externalTrafficPolicy=Local.
+	// Endpoints are chosen based on the first topology key with available backends,
+	// starting at index 0.
+	// If this field is specified and all entries have no backends that match
+	// the topology of the client, the service has no backends for that client
+	// and connections should fail.
+	// The special value "*" may be used to mean "any node". This catch-all
```

Comment:
s/any node/any topology

Reply:
btw, this is still unresolved, can we fix in follow-up PR?

Reply:
Thanks for reminding, now it was resovled.